helmfile/helmfile.yaml

```diff
@@ -0,0 +1,26 @@
+repositories:
+  - name: stable
+    url: https://charts.helm.sh/stable
+
+context: {{ requiredEnv "K8S_CLUSTER_NAME"}}
+   
+environments:
+  dev:
+  int:
+  main:
+  sprint:
+  prod-blue:
+  prod-green: 
+
+releases:
+# Single Tier Website
+  - name: {{ requiredEnv "APP_NAME" }}-{{ .Environment.Name }}
+    chart: ../helm/Charts/single-tier-appliation
```

Comment:
I'm assuming you had a talk with Larabie about this one but highlighting that we weren't 100% on the paths so it doesn't get forgotten

Reply:
It is correct since the helm folder had to be created to keep the charts,values and templates together.